x/ibc/02-client/keeper/querier.go

```diff
@@ -0,0 +1,104 @@
+package keeper
+
+import (
+	"fmt"
+
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/ibc/02-client/types"
+)
+
+// NewQuerier creates a querier for the IBC client
+func NewQuerier(k Keeper) sdk.Querier {
```

Comment:
What about the queriers for `clients`, `states`, `paths` and `commitments`? Also do we require some `evidence` querying here?